_eo_021_enemies/orthoblue_dragon.md

```diff
@@ -5,11 +5,19 @@ image: orthoblue_dragon.png
 start_floor: 25
 end_floor: 27
 agro: 'Sight'
+hp: 134940
+attack_damage: 3007
 attack_type: Physical
 vulnerabilities:
   bind: false
   heavy: false
-  sleep: '?'
+  sleep: true
   slow: true
   stun: false
+abilities:
+  - name: Chilling Aspiration
+    potency: 1000
+    type: Magic
+    description: 'telegraphed line AoE; inflicts frostbite (magic DoT potency
+    40, 20s)
```

Comment:
Missing closing quote. Btw, I have given up on using single quotes everywhere and have been often using doubles lately, mainly to avoid having to use double-singles. Feel free to be inconsistent!

Reply:
Oops, fixed. Also good to know that double quotes work - I won't go back and change these but may switch in the future!